packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -73,12 +73,17 @@ AppConfig.restore = () => {}
 AppConfig.extraGetPropsArgs = () => {}
 AppConfig.freeze = () => {}
 
-// Recommended settings for PWA-Kit usage.
+const isServerSide = typeof window === 'undefined'
+// Recommended settings for PWA-Kit usages.
 const options = {
     queryClientConfig: {
         defaultOptions: {
             queries: {
-                retry: false
+                retry: false,
+                // Avoid double requests during hydration
```

Comment:
Are our partner developers going to understand this comment? Will they immediately be able to make the connection that setting the stale time will effectively avoid requests during hydration? My vote would be to leave this out.

Reply:
@bendvc if we take out the comment, I guess I should delete the other comment below it too? If so, how would developers know why we recommend this set of default options?

If the comment is not clear, I'd rather revise it to be clearer instead.

Reply:
Is this the only reason that we need to set `staleTime`? Agreed with Ben that this comment might cause confusion because some developers might not fully understand how the requests work on the server and client. By definition, staleTime is a config to tell react-query how long the data goes from active to stale, which decides when a network call is needed, and by default, react-query sets it to 0.
If we need to use it as a solution for our problem, we might want to give it a little bit more context in the comment.